File: WindowsApplication1/Modules/Class4.vb
Imports System
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms
Imports System.Data
Imports System.Configuration
Imports System.Resources
Imports System.Drawing
Imports System.Drawing.Drawing2D
Imports System.IO
Imports System.Drawing.Printing
Public Class Class4
    Shared Sub CONNIC()
        If cn.State = ConnectionState.Closed Then
            cn.Open()
        Else
            'CALL CLASS4.CONNIC()
        End If
    End Sub
End Class

File: WindowsApplication1/Modules/Class4.cs
using System.Data;

namespace MCS
{
    public class Class4
    {
        public static void CONNIC()
        {
            if (MCS.Module1.cn.State == ConnectionState.Closed)
            {
                MCS.Module1.cn.Open();
            }
            else
            {
                // CALL CLASS4.CONNIC()
            }
        }
    }
}